doc/Extensions/Rancid.md

```diff
@@ -130,8 +130,9 @@ Add Rancid into LibreNMS config.php:
 ```php
 ### Rancid
 $config['rancid_configs'][]             = '/var/lib/rancid/librenms/configs/';
+$config['rancid_repo_type']             = 'svn';
```

Comment:
I'd add a comment to the end of this line showing the available options like `//svn or git`